app/controllers/concerns/two_factor_authenticatable.rb

```diff
@@ -254,6 +252,10 @@ def display_phone_to_deliver_to
     end
   end
 
+  def decorated_user
+    current_user.decorate
+  end
```

Comment:
I was getting a reek error here, I believe, so I created a little private method. This PR was as refactor, just trying to have a more consistent approach to decorating users (previously we were using a combination of a global helper plus calling `current_user.decorate`) -- I could have been convinced to go either way, just decided to go this way because it seemed nice to avoid the global helper.